applications/GeoMechanicsApplication/custom_elements/U_Pw_small_strain_interface_element.cpp

```diff
@@ -693,27 +704,20 @@ void UPwSmallStrainInterfaceElement<TDim,TNumNodes>::
 
         InterfaceElementVariables Variables;
         const PropertiesType& rProp = this->GetProperties();
-        const GeometryType& rGeom = this->GetGeometry();
-        const unsigned int NumGPoints = rGeom.IntegrationPointsNumber(mThisIntegrationMethod);
-        std::vector<double> GPValues(NumGPoints);
 
         this->InitializeElementVariables(Variables,
-            rGeom,
```

Comment:
usually we use a prefix "r" to indicate that the variable is a reference. In the same function we have rProp. Why did you remove "r"?

Reply:
I will change it back